amr-wind/incflo.cpp

```diff
@@ -11,7 +11,52 @@
 
 #include "AMReX_ParmParse.H"
 
+#include "helics/cpp98/CombinationFederate.hpp"
+#include "helics/cpp98/helics.hpp"
+#include "helics/cpp98/Federate.hpp"
+
+
+#include <sstream>
+#include <iostream>
+#include <string>
+
+using namespace std; 
+
+
 using namespace amrex;
+// /*********************** HELICS **********************/
+
+// /*
+HelicsTime currenttime = 0.0;
+std::string fedinitstring = "--federates=1";
+double deltat = 0.01;
+helicscpp::Input sub;
+helicscpp::Publication pub;
+helicscpp::FederateInfo fi("zmq");
+// Create value federate 
+helicscpp::CombinationFederate vfed("Test receiver Federate",fi);
+
+
```

Comment:
we should probably make a little helics class and then connect it to CFDSim... that way we can do things like `m_sim.helics().vfed.setProperty(blah,1.0)`